include/openspace/camera/camera.h

```diff
@@ -93,6 +94,7 @@ class Camera {
     float sinMaxFov() const;
     SceneGraphNode* parent() const;
     float scaling() const;
+    float atmosphereDimming() const;
```

Comment:
Maybe `atmosphereDimmingFactor`?  Would be nice to have a one-line description of what it is, too